src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -110,10 +115,21 @@ <h3>Launch with</h3>
               ><img src="../assets/images/thirdparty/cavatica.png" /> Cavatica
             </a>
           </div>
+
+          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'gxformat2'">
+            <app-multi-cloud-launch
+              [cloudInstances]="cloudInstances"
+              [usersCloudInstances]="usersCloudInstances"
+              [languagePartner]="partner.GALAXY"
+              [launchUrlAppend]="'%23workflow/' + workflow?.full_workflow_path"
+              [user]="user"
+            ></app-multi-cloud-launch>
+          </div>
+
           <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'NFL'">
             <a
               mat-raised-button
-              [matTooltip]="!(hasContent$ | async) ? 'The WDL has no content.' : 'Export this workflow to Nextflow Tower'"
+              [matTooltip]="!(hasContent$ | async) ? 'The Nextflow has no content.' : 'Export this workflow to Nextflow Tower'"
```

Comment:
`The Nextflow` sounds odd to me. I'd say "The Nextflow workflow" or "The workflow".